code/game/objects/items/religion.dm

```diff
@@ -1,27 +1,148 @@
+			if(has_job_loyalties && H.mind.assigned_role in job_loyalties)
+				inspired += H
+			else if(has_role_loyalties && H.mind.special_role in role_loyalties)
+				inspired += H
+		else if(check_inspiration(H))
+			inspired += H
+
+	for(var/V in inspired)
+		var/mob/living/carbon/human/H = V
+		if(H != user)
+			to_chat(H, "<span class='notice'>Your confidence surges as [user] flourishes their [name]!</span>")
+		inspiration(H)
+		special_inspiration(H)
+
+/obj/item/banner/proc/check_inspiration(mob/living/carbon/human/H) //Banner-specific conditions for being eligible
+	return
+
+/obj/item/banner/proc/inspiration(mob/living/carbon/human/H)
+	H.adjustBruteLoss(-15)
+	H.adjustFireLoss(-15)
+	H.AdjustStun(-40)
+	H.AdjustKnockdown(-40)
+	H.AdjustUnconscious(-40)
+	playsound(H, 'sound/magic/staff_healing.ogg', 25, FALSE)
+
+/obj/item/banner/proc/special_inspiration(mob/living/carbon/human/H) //Any banner-specific inspiration effects go here
+	return
+
+/obj/item/banner/security
+	name = "securistan banner"
+	desc = "The banner of Securistan, ruling the station with an iron fist."
+	icon_state = "banner_security"
+	job_loyalties = list("Security Officer", "Warden", "Detective", "Head of Security")
+	warcry = "EVERYONE DOWN ON THE GROUND!!"
+
+/obj/item/banner/security/mundane
+	inspiration_available = FALSE
+
+/obj/item/banner/medical
+	name = "meditopia banner"
+	desc = "The banner of Meditopia, generous benefactors that cure wounds and shelter the weak."
+	icon_state = "banner_medical"
+	job_loyalties = list("Medical Doctor", "Chemist", "Geneticist", "Virologist", "Chief Medical Officer")
+	warcry = "No wounds cannot be healed!"
+
+/obj/item/banner/medical/mundane
+	inspiration_available = FALSE
+
+/obj/item/banner/medical/check_inspiration(mob/living/carbon/human/H)
+	return H.stat //Meditopia is moved to help those in need
+
+/obj/item/banner/medical/special_inspiration(mob/living/carbon/human/H)
+	H.adjustToxLoss(-15)
+	H.setOxyLoss(0)
+	H.reagents.add_reagent("inaprovaline", 5)
+
+/obj/item/banner/science
+	name = "sciencia banner"
+	desc = "The banner of Sciencia, bold and daring thaumaturges and researchers that take the path less traveled."
+	icon_state = "banner_science"
+	job_loyalties = list("Scientist", "Roboticist", "Research Director")
+	warcry = "For Cuban Pete!"
+
+/obj/item/banner/science/mundane
+	inspiration_available = FALSE
+
+/obj/item/banner/science/check_inspiration(mob/living/carbon/human/H)
+	return H.on_fire //Sciencia is pleased by dedication to the art of Toxins
+
+/obj/item/banner/cargo
+	name = "cargonia banner"
+	desc = "The banner of the eternal Cargonia, with the mystical power of conjuring any object into existence."
+	icon_state = "banner_cargo"
+	job_loyalties = list("Cargo Technician", "Shaft Miner", "Quartermaster")
+	warcry = "Hail Cargonia!"
+
+/obj/item/banner/cargo/mundane
+	inspiration_available = FALSE
+
+/obj/item/banner/engineering
+	name = "engitopia banner"
```

Comment:
atmosia, not engitopia!!

Reply:
Atmosia and Enginistan formed a peace treaty and united as Engitopia. The Atmosia Declaration of Independence poster you can find proves that its independence is part of history now.